### PR TITLE
Various minor tweaks

### PR DESCRIPTION
**Pull request by ajrgrubbs**
- Changed process creation node color to 5C secondary brand color
- Moved domain into config file
- Can't drag progress bar below 0
- Sped up CSS link drawing animation

### CODE REVIEW

File: vee/static/treeplayer.js [hunk 1]
```diff
@@ -221,15 +221,19 @@ class TreePlayer {
     d3bar.on('mousedown', function() {
       var e = d3.event;
       var rect = this.getBoundingClientRect();
-      var percent = Math.min((e.x - rect.left) / rect.width, 1);
+      var percent = (e.x - rect.left) / rect.width;
+      percent = Math.min(percent, 1); // Set upper bound
+      percent = Math.max(percent, 0); // Set lower bound
```
Comment (adamyala) [inline vee/static/treeplayer.js]: Why set a variable then set to something else immediately after? Is this a JS thing?

Comment (ajrgrubbs) [inline vee/static/treeplayer.js]: Nah it's a readability thing. Before I broke it out into the three lines, it looked like
`var percent = Math.max(Math.min((e.x - rect.left) / rect.width, 1), 0)` and that sucks.

Comment (csavage1994) [inline vee/static/treeplayer.js]: If this is going to stay in there, you should probably include a comment explaining why you chose to do it this way and what it was before for reference

Comment (csavage1994) [inline vee/static/treeplayer.js]: If this is going to stay in there, you should add a comment explaining why you're doing it this way and what is was before to add some context

**Comment by csavage1994** [comment]
👍

**Comment by vforgione** [comment]
👍

**Pull request merged**